rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/report/SW360ReportController.java

```diff
@@ -103,12 +105,15 @@ public void getProjectReport(
             @RequestParam(value = "variant", required = false) String variant,
             @Parameter(description = "Template for generating report. Can be supplied with modules [" + LICENSE_INFO + ", " + EXPORT_CREATE_PROJ_CLEARING_REPORT + "]")
             @RequestParam(value = "template", required = false, defaultValue = "") String template,
-            @Parameter(description = "The external Ids of the project. Can be supplied with modules [" + LICENSE_INFO + ", " + EXPORT_CREATE_PROJ_CLEARING_REPORT + "]", example = "376577")
+            @Parameter(description = "The external Ids of the project. External Ids can be supplied multiple with comma separated. " +
+                    "Can be supplied with modules [" + LICENSE_INFO + ", " + EXPORT_CREATE_PROJ_CLEARING_REPORT + "]", example = "376577")
```

Comment:
If we are updating the docs, lets update the example value as well :-)
```suggestion
            @Parameter(description = "The external Ids of the project. External Ids can be supplied multiple with comma separated. " +
                    "Can be supplied with modules [" + LICENSE_INFO + ", " + EXPORT_CREATE_PROJ_CLEARING_REPORT + "]", example = "376577,12345")
```